### PR TITLE
add version command

### PR DESCRIPTION
**Pull request by zacharym**
@jsingle cc @thieman

### CODE REVIEW

File: dusty/commands/version.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from ..constants import VERSION
+from ..log import log_to_client
+
+def version():
+    log_to_client('Dusty version: {}'.format(VERSION))
```
Comment (jsingle) [inline dusty/commands/version.py]: Let's make it clear that it's the *daemon* version

**Comment by jsingle** [comment]
I'm not the one with the vision for the separate `dusty -v` and `dusty version` commands.  Seems to me like we should print out the client version as well in this command

**Comment by jsingle** [comment]
Also it would be kind of funny to run `dusty version` and get a RuntimeError saying the client and daemon versions are different lol.  Not sure if that's worth addressing

File: dusty/commands/version.py (hunk 1 above)
Comment (jsingle) [inline dusty/commands/version.py]: DAEEEMON

Comment (thieman) [inline dusty/commands/version.py]: +1

**Pull request merged**